db/init_db.js

```diff
@@ -1,25 +1,477 @@
 // code to build and initialize DB goes here
 const {
   client
-  // other db methods 
+  // createUser, createProducts, etc. 
 } = require('./index');
 
 async function buildTables() {
   try {
     client.connect();
+      await client.query(`
+        DROP TABLE IF EXISTS users, products, orders, order_products;
+      `);
+  
+      await client.query(`
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY, 
+        firstname VARCHAR(255) NOT NULL, 
+        lastname VARCHAR(255) NOT NULL, 
+        email VARCHAR(255) UNIQUE NOT NULL, 
+        "imgURL" DEFAULT VALUE, 
```

Comment:
let's get the actual default VALUE in here to start, something like:
`"imgURL" DEFAULT default_img.jpg,`